grakn-test/src/test/java/ai/grakn/test/migration/MigratorTestUtils.java

```diff
@@ -123,46 +123,50 @@ public static Resource getResource(GraknGraph graph, Thing thing, TypeLabel labe
     /**
      * Check that the pet graph has been loaded correctly
      */
-    public static void assertPetGraphCorrect(GraknGraph graph){
-        Collection<Entity> pets = graph.getEntityType("pet").instances();
-        assertEquals(9, pets.size());
+    public static void assertPetGraphCorrect(GraknSession session){
+        try(GraknGraph graph = session.open(GraknTxType.READ)) {
+            Collection<Entity> pets = graph.getEntityType("pet").instances();
+            assertEquals(9, pets.size());
 
-        Collection<Entity> cats = graph.getEntityType("cat").instances();
-        assertEquals(2, cats.size());
+            Collection<Entity> cats = graph.getEntityType("cat").instances();
```

Comment:
On my laptop they show up as spaces :( Swear I have not gone back to the dark side